semver missing from a clean build

Steps to replicate:

1. use serverless to create a new project
2. `yarn add serverless-esbuild`
3. `serverless package`

Expected: it should work

Actual:

```
Error: Cannot find module 'semver'
  Require stack:
  - /.../scheduler/sync/node_modules/serverless-esbuild/dist/pack.js
  - /.../scheduler/sync/node_modules/serverless-esbuild/dist/index.js
  - /snapshot/serverless/lib/classes/PluginManager.js
  - /snapshot/serverless/lib/Serverless.js
  - /snapshot/serverless/scripts/serverless.js
  - /snapshot/serverless/bin/serverless.js
```

--- a/package.json
+++ b/package.json
@@ -48,6 +48,7 @@
     "@types/jest": "^26.0.14",
     "@types/node": "^12.12.38",
     "@types/ramda": "^0.27.6",
+    "@types/semver": "^7.3.8",
     "@types/serverless": "^1.78.25",
     "@typescript-eslint/eslint-plugin": "^4.2.0",
     "@typescript-eslint/parser": "^4.2.0",
@@ -70,6 +71,7 @@
     "globby": "^11.0.1",
     "p-map": "^4.0.0",
     "ramda": "^0.27.0",
+    "semver": "^7.3.5",
     "string.prototype.matchall": "^4.0.4"
   },
   "lint-staged": {

